troubleshooting/known_issues/nmn_dns_r_records_missing.md

```diff
@@ -0,0 +1,221 @@
+
+### 1: Get the version of the `csm-packages` from compute node BOS session template
+
+Example:
+
+Find the session template name (`ncn#`):
+
+```bash
+cray bos sessiontemplates list | grep compute-*
+```
+
+Example output:
+
+```toml
+name = "compute-25.3.0-alpha2.x86_64-161rc7"
+```
+
+Using the `sessiontemplate` name from the previous command, find the `configuration` name (`ncn#`):
+
+```bash
+cray bos sessiontemplates describe compute-25.3.0-alpha2.x86_64-161rc7  --format json
+```
+
+Example output:
+
+```json
+{
+    "cfs": {
+        "configuration": "compute-25.3.0-alpha2-161rc7"
+    }
+}
+```
+
+Use the `configuration` value to describe the configuration (`ncn#`):
+
+```bash
+cray cfs configurations describe compute-25.3.0-alpha2-161rc7 --format json
+```
+
+Example output:
+
+```json
+{
+    "lastUpdated": "2025-03-12T07:25:18Z",
+    "layers": [
+        {
+          "cloneUrl": "https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git",
+          "commit": "eaa4ee6948961592b0fa279ae775326ad63eb875",
+          "name": "csm-packages-1.6.1-rc.7",
+          "playbook": "csm_packages.yml"
+        }
+    ]
+}
+```
+
+The `name` from the describe above identifies the product catalog. Use the version after `csm-packages-` in the next step.
+
+### 2: Get the corresponding `csm-config` branch (@VCS) from product catalog given `csm-packages-*` version found from `Step-1`
+
+Example (`ncn-#`):
+
+```bash
+kubectl get cm -n services cray-product-catalog -o yaml | yq - r 'data.csm' | grep ^1.6.0-rc.4: -A 10
+```
+
+Example output:
+
+```yaml
+1.6.1-rc.7:
+  configuration:
+    clone_url: https://vcs.cmn.drax.hpc.amslabs.hpecorp.net/vcs/cray/csm-config-management.git
+    commit: eaa4ee6948961592b0fa279ae775326ad63eb875
+    import_branch: cray/csm/1.28.0
+```
+
+The `import_branch` from above output will be used below.
+
+### 3: Log into VCS and clone `csm-config-management.git` @ VCS
+
+Example (`ncn#`):
+
+```bash
+USERNAME=$( kubectl get secrets -n services vcs-user-credentials -o json | jq -r .data.vcs_username | base64 -d  )
+PSWD=$( kubectl get secrets -n services vcs-user-credentials -o json | jq -r .data.vcs_password | base64 -d  )
+git clone https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git
+```
+
+**Note:** use above $USERNAME and $PSWD for VCS login
+
+### 4: Apply workaround from `import_branch` found in `step-2`
+
+Example (`ncn#`):
+
+```bash
+cd csm-config-management
+git checkout cray/csm/1.28.0
+```
+
+Copy scripts `roles/csm.sbps.dns_srv_records/files/sbps_get_host_hsn_nmn.sh` and `roles/csm.sbps.dns_srv_records/files/sbps_dns_srv_records.sh` from this branch 
+to any of the worker node followed by executing below commands:
+
+```bash
+# chmod +x sbps*
+```
+
+```bash
+# ./sbps_get_host_hsn_nmn.sh | ./sbps_dns_srv_records.sh
+```
+
+### 5: After workaround applied
+
+On any worker node, issue one of the below command to see if we are able to see DNS "SRV" "A" records for NMN too

```

Comment:
wording:
- change `the below command` to `the following commands`
- change  `records for NMN too` to `records for the NMN`